samples/photo_album/package.json

```diff
@@ -5,41 +5,41 @@
   "description": "Sample application using Cloudinary's Angular2 SDK",
   "license": "MIT",
   "dependencies": {
-    "@angular/animations": "^4.0.1",
-    "@angular/common": "^4.0.1",
-    "@angular/compiler": "^4.0.1",
-    "@angular/compiler-cli": "^4.0.1",
-    "@angular/core": "^4.0.1",
-    "@angular/forms": "^4.0.1",
-    "@angular/http": "^4.0.1",
-    "@angular/platform-browser": "^4.0.1",
-    "@angular/platform-browser-dynamic": "^4.0.1",
-    "@angular/platform-server": "^4.0.1",
-    "@angular/router": "^4.0.1",
-    "@cloudinary/angular-4.x": "^1.0.0",
+    "@angular/animations": "^5.0.1",
+    "@angular/common": "^5.0.1",
+    "@angular/compiler": "^5.0.1",
+    "@angular/compiler-cli": "^5.0.1",
+    "@angular/core": "^5.0.1",
+    "@angular/forms": "^5.0.1",
+    "@angular/http": "^5.0.1",
+    "@angular/platform-browser": "^5.0.1",
+    "@angular/platform-browser-dynamic": "^5.0.1",
+    "@angular/platform-server": "^5.0.1",
+    "@angular/router": "^5.0.1",
+    "@cloudinary/angular-5.x": "file:../../cloudinary-angular-5.x-v1.0.0.tgz",
```

Comment:
It's temporary, since SDK version 5 is not in NPM yet.
Referencing a non-existing file won't build (and I know that the build is borken while referencing a local file), but a missing NPM file will also corrupt `package-lock.json`.

Once the v5 of SDK is published I will update all samples to point to NPM.